static/js/composebox_typeahead.js

```diff
@@ -788,8 +792,18 @@ export function show_flatpickr(element, callback, default_timestamp, options = {
         formatDate: (date) => formatISO(date),
         disableMobile: true,
         onKeyDown: (selectedDates, dateStr, instance, event) => {
+            if (is_numeric_key(event.key)) {
+                // Don't stopPropagation for numeric inputs, let them be handled normally
+                return;
+            }
+
             const hotkey = get_keydown_hotkey(event);
 
+            if (hotkey === "backspace" || hotkey === "delete") {
```

Comment:
Was this supposed to be `hotkey.name`?

Reply:
yes! I thought I'd changed that, thanks for pointing out.

Reply:
#20688